### ISSUE TITLE
overriding fetch()

### ISSUE THREAD

**Issue by buendia** [HenrikJoreteg/Capsule#4]
calling Capsule.model.fetch() leads to an error like this:

A 'url' property or function must be specified

should it be overridden to be more integrated with socket.io, or should this method be removed?

**Comment by HenrikJoreteg** [comment]
Yeah, it should probably just be removed... that's a good point. (sorry i didn't see your comments before, I haven't been getting github notifications when people filed issues, on this project for some reason).
